package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/va-forms-system-core",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
I don't think updating an example would need a version update since it doesn't look like the example code is included in the main export that gets published to npm:

https://github.com/department-of-veterans-affairs/va-forms-system-core/blob/bc30fc8a00d21355cd9cf876530353eab52c3247/src/index.ts#L1-L5

Reply:
Great point, I reverted the update to package.json.